openapi.yaml

```diff
@@ -356,3 +503,202 @@ components:
           type: string
           enum: [positions]
           example: 'positions'
+
+    ShiftListResponse:
+      type: object
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ShiftResponse'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationResponse'
+
+    ShiftResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          example: '1'
+        type:
+          type: string
+          example: 'shifts'
+        attributes:
+          $ref: '#/components/schemas/ShiftAttributes'
+        relationships:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationsRelationship'
+      required:
+        - id
+        - type
+
+    ShiftAttributes:
+      type: object
+      properties:
+        status:
+          type: string
+          example: 'open'
+        starts_at:
+          type: string
+          example: '2020-01-01T06:00:00.000000Z'
+        ends_at:
+          type: string
+          example: '2020-01-01T22:00:00.000000Z'
+        positions:
+          type: array
+          items:
+            type: string
+          example: ['rn']
+        inserted_at:
+          type: string
+          example: '2020-01-01T06:00:00.000000Z'
+        updated_at:
+          type: string
+          example: '2020-01-01T06:00:00.000000Z'
+        rates:
+          type: array
+          items:
+            type: object
+            properties:
+              position:
+                type: string
+                example: 'rn'
+              currency:
+                type: string
+                example: 'USD'
+              bill_rate:
+                type: integer
```

Comment:
[sand] Is there a description attribute or something similar we can include to explicitly indicate how to interpret these rates. Basically make it obvious this is `$60` rather than `$6000`.

Maybe it's not in the OpenAPI spec. Can't remember.

Reply:
Maybe:
```
              bill_rate:
                type: integer
                description: "Decimal value in cents. Example: 6000 = $60.00"
                example: 6000
```